plugin/evm/vm.go

```diff
@@ -239,7 +249,7 @@ type VM struct {
 	// Metrics
 	sdkMetrics *prometheus.Registry
 
-	bootstrapped bool
+	bootstrapped avalancheUtils.Atomic[bool]
```

Comment:
curious what new concurrent access pattern is added for bootstrapped?

Reply:
ahh this was for the locked uptime calculator https://github.com/ava-labs/subnet-evm/pull/1367/files#diff-3be5ea9687f8af021d9baaa6584d174a7975b181b604e1a759e4ad2e40136912R515. I moved those to the separate PR but forgot to revert the changes here.

Reply:
I see... makes sense.